src/core.js

```diff
@@ -822,7 +823,7 @@ $.extend( $.validator, {
 					if ( group ) {
 						$.each( this.groups, function( name, testgroup ) {
 							if ( testgroup === group ) {
-								$( "[name='" + name.replace( /'/g, "\\'" ) + "']", this.currentForm )
+								$( "[name='" + this.escapeCssMeta( name ) + "']", this.currentForm )
```

Comment:
@Arkni I just did some manual testing in a real world app with todays master version. I got errors like

```
TypeError: this.escapeCssMeta is not a function

.prototype.showLabel/<()     jquery.validate.js:927
.each()   jquery.js:2
.prototype.showLabel()    jquery.validate.js:925
.prototype.defaultShowErrors()    jquery.validate.js:833
.prototype.showErrors()   jquery.validate.js:499
.prototype.element()    jquery.validate.js:468
.onfocusout()
```

with a `onfocusout` default setting like

```js
$.validator.setDefaults({
// ...snip
        onfocusout: function(element) {
        	// activate immediate-as-you-type validation, by overriding default behaviour
            var isValid = this.element(element);
            
        	// delete the wrapping "field-with-errors" element, when fields get valid
            if (isValid) {
            	var $el = $(element);
            	if ($el.parent().is(".field-with-errors")) {
            		$el.unwrap();
            	}
            }
        },
// snap...
```

atm I dont have time to investigate it furhter. just to let you know in the meantime.